protobuf/drand/api.proto

```diff
@@ -86,6 +86,8 @@ message PublicRandResponse {
     // randomness is simply there to demonstrate - it is the hash of the
     // signature. It should be computed locally.
     bytes randomness = 4;
+    // timestamp of the round
+    int64 timestamp = 5;
```

Comment:
protobuf has some timestamp method I think. It is important that this appears as a date that can be read by a human (usually RFC 3339 )